docs/guide/the-user-interface.md

```diff
@@ -76,12 +76,10 @@ Checking the video caption option will display the acceleration value in the top
 
 %end
 
-- The **Export HTML5 Scene...** item allows you to export the current world as an interactive 3D ".html" file.
-You can get more information about this topic in [this section](web-scene.md).
-
-- ![](images/animation-button.png =26x26) The **Make HTML5 Animation...** item allows you to record a simulation as a 3D animation and publish it on a HTML5 web page.
-Once the animation recording is started, this item is changed to `Stop HTML5 Animation...` and can be used to stop the animation recording.
-You can get more information about this topic in [this section](web-animation.md).
+- ![](images/share-button.png =26x26) The **Share...** item allows you to export the current world as an interactive 3D scene (more information about this topic in [this section](web-scene.md)), or to record a simulation as a 3D animation and publish it on a HTML5 web page.
+For both options, it is possible to save the result locally or to upload it automatically to webots.cloud where it can be easily shared and seen by other people.
```

Comment:
```suggestion
For both options, it is possible to save the result locally or to upload it automatically to [webots.cloud](https://webots.cloud) where it can be easily shared and seen by other people.
```